ClosedXML.Tests/Excel/Loading/LoadingTests.cs

```diff
@@ -32,6 +32,19 @@ public void CanSuccessfullyLoadLOFiles(string file)
             TestHelper.LoadFile(file);
         }
 
+#if !_NET40_
+        // .NET 4.0 does not recognise the malformed mailto URI in the file
+        [TestCaseSource(nameof(MalformedFiles))]
+        public void CanSuccessfullyLoadMalformedFilesWithRemoveMalformedHyperlinksRelationshipErrorHandlerFactory(string file)
```

Comment:
Perhaps, we should also have a test checking that opening these files _without_ `RemoveMalformedHyperlinksRelationshipErrorHandlerFactory` fails. If that test starts to fail one day, that will be an indicator for us that the factory is not needed anymore and we can consider removing (deprecating) it.